apps/jobbrowser/src/jobbrowser/templates/job_browser.mako

```diff
@@ -3026,6 +3026,25 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
         });
       };
 
+      self._rewriteKnoxUrls = function (data) {
+        if (data.app.type === 'SPARK') {
+          data.app.properties.metadata.forEach(function (item) {
+            if (item.name === 'trackingUrl') {
+              /*
+                Rewrite tracking url
+                Sample trackingUrl: http://<yarn>:8088/proxy/application_1652826179847_0003/
+                Knox URL: https://<knox-base>/yarn/cluster/app/application_1652826179847_0003
+              */
+              const matches = item.value.match('(application_[0-9_]+)');
```

Comment:
Nit: Switch const to var, also line 3040. Likely not an issue as most browsers support const nowadays but more for consistency.

Reply:
Swtiched to `var`. I have limited javascript experience but had recently done some typescript too and it is coming from there.